chromium_src/third_party/blink/renderer/modules/eventsource/event_source.cc

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "third_party/blink/renderer/modules/eventsource/event_source.h"
+
+#include "brave/third_party/blink/renderer/core/farbling/brave_session_cache.h"
+#include "third_party/blink/public/common/features.h"
+#include "third_party/blink/public/common/scheme_registry.h"
+#include "third_party/blink/public/platform/web_content_settings_client.h"
+#include "third_party/blink/renderer/core/dom/document.h"
+#include "third_party/blink/renderer/core/frame/dom_window.h"
+#include "third_party/blink/renderer/core/streams/readable_byte_stream_controller.h"
+#include "third_party/blink/renderer/platform/loader/fetch/resource_error.h"
+
+#define ConnectTimerFired(...) ConnectTimerFired_ChromiumImpl(__VA_ARGS__)
```

Comment:
I wasn't sure how this override was working, so I had to test it manually. It turns out this `#define` replaces only function implementation, but doesn't replace `&EventSource::ConnectTimerFired` which is what we want. TIL!

Reply:
This was indeed lucky because I was struggling to find a unique member function to replace.